pandas/tests/frame/test_arithmetic.py

```diff
@@ -2040,3 +2040,13 @@ def _constructor_sliced(self):
 
     result = sdf + sdf
     tm.assert_frame_equal(result, expected)
+
+
+def test_frame_sub_dtype():
+    # GH 32822
+    s = Series([1, 2, np.nan], dtype='Int64')
+    t = Series([1, 2, 3], dtype='Int64')
+
+    expected = (s - t).dtype
```

Comment:
Could you please validate the whole objects with tm.assert_series_equals and please use multiletter variables